css/search.css

```diff
@@ -0,0 +1,340 @@
+  background-image: url(../img/btn-wishlist-hover.png);
+}
+
+.search-container--view {
+  display: flex;
+  border-width: 1px 0 1px 0;
+  border-style: solid;
+  border-color: #dfdfdf;
+  margin: 10px 0;
+}
+
+.container--view {
+  display: flex;
+  padding: 10px 0;
+}
+
+.view-button {
+  color: #808080;
+  background-color: #d4d4d4;
+  margin-right: 10px;
+  cursor: pointer;
+  padding: 10px;
+}
+
+.view-button--active {
+  background-color: #fca53c;
+  color: #fff;
+  cursor: default;
+  pointer-events: none;
+}
+
+.view-button:hover {
+  background-color: #fca53c;
+  color: #fff;
+}
+
+.conteiner--drinksGroups {
+  margin-left: auto;
+  display: flex;
+  text-align: center;
+}
+
+.group-drink {
+  display: flex;
+  box-sizing: border-box;
+  align-items: center;
+  justify-content: center;
+  padding: 10px;
+  margin: 10px 5px;
+  width: 32px;
+  border-width: 1px;
+  border-style: solid;
+  border-color: #dfdfdf;
+  color: #000;
+  cursor: pointer;
+}
+
+.group-drink--active {
+  color: #fca53c;
+  cursor: default;
+}
+
+.group-drink:hover {
+  color: #fca53c;
+}
+
+.unfulfilled-condition {
+  display: flex;
+  padding: 15px;
+  border-radius: 15px;
+  margin-bottom: 30px;
+  justify-content: space-between;
+  background-color: #ffced4;
+  color: #d04a5b;
+  cursor: default;
+}
+
+.unfulfilled-condition-message {
+  height: fit-content;
+  margin: auto 0px;
+}
+
+.clouse-message {
+  font-size: 2rem;
+  cursor: pointer;
+}
+
+.clouse-message:hover {
+  color: #808080;
+}
+
+.not-match-any {
+  padding: 30px 0 15px;
+  margin: 0 15px;
+}
+
+@media (min-width: 480px) {
+  .drink--table.drink--wrapper {
+    padding: 15px;
+  }
+
+  .drink--table .drink-name {
+    display: block;
+    font-size: 1.4rem;
+    height: 4.2rem;
+  }
+
+  .drink--table .view-button {
+    display: none;
+  }
+
+  .drink--table .drink-description {
+    display: none;
+  }
+
+  .drink--table .drink {
+    display: block;
+    box-sizing: border-box;
+    border-width: 1px;
+    border-style: solid;
+    border-color: #dfdfdf;
+    padding: 10px 15px;
+    height: auto;
+  }
+
+  .drink--table .drink-img {
+    width: 100%;
+    border-width: 0;
+  }
+
+  .drink--table .drink-info {
+    width: 100%;
+    padding: 0;
+  }
+
+  .drink--table .container-litrage-and-prise {
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: baseline;
+  }
+}
+
+@media (min-width: 760px) {
+  .drink--table.drink--wrapper {
+    width: 33.3333%;
+    padding: 10px 15px;
+  }
+}
+
+@media (min-width: 480px) and (max-width: 760px) {
+  .drink--table.drink--wrapper {
+    width: 50%;
+  }
+}
+
+@media (max-width: 480px) {
+  .drink-name {
+    display: block;
+    font-size: 1.4rem;
+    height: 4.2rem;
+  }
+
+  .view-button {
+    display: none;
+  }
+
+  .drink-description {
+    display: none;
+  }
+
+  .drink {
+    display: block;
+    box-sizing: border-box;
+    border-width: 1px;
+    border-style: solid;
+    border-color: #dfdfdf;
```

Comment:
border
if you don't understand what i mean
just write
border: <width> <style> <color>
border: 1px solid #dfdfdf;